### PR TITLE
[4.0] RTL: correcting some fields display

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes 
Adding specific RTL scss to display correctly some fields. 
Adding text-align left for redirects URLs fields

### Testing Instructions
Install a multilingual site. Install Persian language.
Load the Article edit page, Publishing tab as this is a good example of the corrections.

Patch and use npm
Look again at that page, play reducing the window to mobile size.

Look also at other admin pages

### Before patch
Screen Shot 2019-11-03 at 16 26 59

### After patch

Screen Shot 2019-11-03 at 16 17 13

### CODE REVIEW

File: administrator/templates/atum/scss/vendor/bootstrap/_form.scss [hunk 1]
```diff
@@ -23,19 +23,36 @@ label {
...
     text-align: left;
-	@include media-breakpoint-down(sm)
-	{
-		float: none;
-		width: 100%;
-	}
+
+    [dir=rtl] & {
+      float: right;
+      text-align: right;;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/vendor/bootstrap/_form.scss]: `;;` > `;`

File: administrator/templates/atum/scss/vendor/bootstrap/_form.scss [hunk 2]
```diff
@@ -23,19 +23,36 @@ label {
...
+      margin-left: 0;
+    }
+
+    @include media-breakpoint-down(sm)
+    {
+      margin-left: 0;
+
+      [dir=rtl] & {
+        margin-right: 0;
+        margin-left: 0;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/vendor/bootstrap/_form.scss]: Don't think you need a left margin if it's already defined as `0` above

File: administrator/templates/atum/scss/vendor/bootstrap/_form.scss (hunk 1 above)
Comment (infograf768) [reply administrator/templates/atum/scss/vendor/bootstrap/_form.scss]: done

File: administrator/templates/atum/scss/vendor/bootstrap/_form.scss (hunk 2 above)
Comment (infograf768) [reply administrator/templates/atum/scss/vendor/bootstrap/_form.scss]: Taking it off. Thanks.

**Comment by infograf768** [comment]
@wilsonge
This should be OK to go now.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**